tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -0,0 +1,469 @@
+        assert self.storage_cluster_obj.patch(
+            resource_name="ocs-storagecluster",
+            params=self.nfs_spec_disable,
+            format_type="merge",
+        ), "storagecluster.ocs.openshift.io/ocs-storagecluster not patched"
+
+        # Enable ROOK_CSI_ENABLE_NFS via patch request
+        assert self.config_map_obj.patch(
+            resource_name="rook-ceph-operator-config",
+            params=self.rook_csi_config_disable,
+            format_type="merge",
+        ), "configmap/rook-ceph-operator-config not patched"
+
+        # Delete the nfs StorageClass
+        cmd_delete_nfs_sc = "delete sc " + self.nfs_sc
+        self.storage_cluster_obj.exec_oc_cmd(cmd_delete_nfs_sc)
+
+        # Delete CephNFS
+        cmd_delete_cephnfs = "delete CephNFS ocs-storagecluster-cephnfs"
+        self.storage_cluster_obj.exec_oc_cmd(cmd_delete_cephnfs)
+
+        # Delete nfs-ganesha pod
+        pod_objs = pod.get_all_pods(
+            namespace=self.namespace, selector=["rook-ceph-nfs"]
+        )
+        for pod_obj in pod_objs:
+            pod_obj.delete()
+            pod_obj.ocp.wait_for_delete(
+                pod_obj.name, 180
+            ), f"Pod {pod_obj.name} is not deleted"
+
+    # def teardown(self):
+    #     """
+    #     Delete cephnfs CR,ocs nfs service and nfs storageclass
+    #     Remove test_nfs folder
+    #     """
+    #     exec_cmd(cmd="rm -rf "+ self.test_folder)
+    #     self.nfs_disable()
+
+    def test_nfs_not_enabled_by_default(
+        self,
+    ):
+        """
+        This test is to validate nfs feature is not enabled by default for  ODF(4.11) clusters
+
+        Steps:
+        1:- Check cephnfs resources not available by default
+
+        """
+        # Checks cephnfs resources not available by default
+        cephnfs_resource = self.storage_cluster_obj.exec_oc_cmd("get cephnfs")
+        if cephnfs_resource is None:
+            log.info(f"No resources found in openshift-storage namespace.")
+
+    def test_nfs_feature_enable(
+        self,
+    ):
+        """
+        This test is to validate nfs feature enable after deployment of  ODF(4.11) cluster
+
+        Steps:
+        1:- Check cephnfs resources not available by default
+        2:- Enable nfs feature for storage-cluster
+        3:- Check cephnfs resource running
+
+        """
+        # Checks cephnfs resources not available by default
+        cephnfs_resource = self.storage_cluster_obj.exec_oc_cmd("get cephnfs")
+        if cephnfs_resource is None:
+            log.info(f"No resources found in openshift-storage namespace.")
+
+        # Enable nfs feature for storage-cluster
+        self.nfs_enable()
+
+        # Check cephnfs resource running
+        cephnfs_resource_status = self.storage_cluster_obj.exec_oc_cmd(
+            "get CephNFS ocs-storagecluster-cephnfs --output jsonpath='{.status.phase}'"
+        )
+        assert cephnfs_resource_status == "Ready"
+
+        self.nfs_disable()
```

Comment:
I think it would be better to move those into an NFS related utils file

Reply:
Addressed in latest commit.